great_expectations/data_context/store/tuple_store_backend.py

```diff
@@ -257,7 +257,23 @@ def list_keys(self, prefix=()):
                     key_list.append(key)
 
         return key_list
+   
     
+    def crmdir(self, mroot, curpath):
+        try:
+            while not os.listdir(curpath) and os.path.exists(curpath):
+                d2=curpath
+                f2=os.path.dirname(d2)
+                if os.path.isdir(curpath):
+                    shutil.rmtree(curpath)
+                else:
+                    os.remove(curpath)
```

Comment:
Not sure this if/else is necessary since os.listdir(curpath) above would have thrown exception if curpath wasn't a dir.

Reply:
If there's ever anything left behind under static etc. Let's say user has some local file that we didn't put there.  catch has NotADirectoryError.  I moved the file check before dir check.